data-reconciliation-app/governance/scripts/setup_governance.sh

```diff
@@ -160,6 +160,14 @@ $root_dir/scripts/submit_proposal.sh --network-url  ${server} \
   --proposal-file $certs/network_open_proposal.json --service-cert $certs/service_cert.pem \
   --signing-cert $certs/member0_cert.pem --signing-key $certs/member0_privk.pem
 
+##############################################
+# Enable Jwt authentication
+##############################################
+echo "Enable Jwt authentication: submit proposal to network and vote as accepted"
+$root_dir/scripts/submit_proposal.sh --network-url  ${server} \
+  --proposal-file $PWD/workspace/configs/set_jwt_issuer_ms_proposal.json --service-cert $certs/service_cert.pem \
+  --signing-cert $certs/member0_cert.pem --signing-key $certs/member0_privk.pem
```

Comment:
nit suggestion for consistency and readability: use `$app_dir` (which already points to `PWD`)

```suggestion
$root_dir/scripts/submit_proposal.sh --network-url  ${server} \
  --proposal-file ${app_dir}/workspace/configs/set_jwt_issuer_ms_proposal.json --service-cert $certs/service_cert.pem \
  --signing-cert $certs/member0_cert.pem --signing-key $certs/member0_privk.pem
```

Reply:
Done